Source/SwiftyDropbox/Shared/Handwritten/ReconnectionHelpers+Handwritten.swift

```diff
@@ -24,9 +24,85 @@ protocol PersistedRequestInfoBaseInfo {
     var clientProvidedInfo: String? { get set }
 }
 
-extension ReconnectionHelpers {
-    private static let Separator = "#?///?#"
+private let Separator = "#?///?#"
```

Comment:
Should this be in the default extension of ReconnectionHelpersShared instead of global space?

Reply:
I wish I could but you can't put a stored property there

Reply:
I made a small Constants struct to get it out of the global space